### ISSUE TITLE
game exited after loading "Deadly Grotto"

### ISSUE THREAD

**Issue by frankstein-1** [openclonk/openclonk#9]
Hello 
I use Linux Mint 17.1 rebecca, my  compiler : 
```bash
gcc --version
gcc (Ubuntu 4.8.4-2ubuntu1~14.04) 4.8.4
Copyright (C) 2013 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```
I installed the game from the source code, available in download page, using "cmake && make all" 
The game exited after loading the mission  "Deadly Grotto"
Before launching this mission I played with other. the rest of log can be found on http://paste.debian.net/343388/ 
My player was deactivated.
```bash
[20:23:13] Player Neuling removed.
[20:23:13] Game cleared.
[20:23:13] Loading graphics...
[20:23:13] Music File not found: /usr/local/share/games/openclonk/Music.ocg
[20:23:32] Scenario: /home/user/openclonk-6.1-src/planet/Missions.ocf/DeadlyGrotto.ocs
[20:23:32] Scenario uses specified object definitions.
[20:23:32] Loading extras...
[20:23:32] Loading graphics...
[20:23:33] Loading object definitions:
[20:23:33] Objects.ocd...
[20:23:36]    292 definitions loaded.
[20:23:36] DeadlyGrotto.ocs...
[20:23:36]    3 definitions loaded.
[20:23:36] C4AulScriptEngine linked - 53674 lines, 0 warnings, 0 errors
[20:23:36] Texture table holds 32 entries.
[20:23:39] 55 textures loaded.
[20:23:39] 22 materials loaded.
[20:23:40] Creating landscape...
[20:23:41]    gl: Material textures too large, GPU only supports 256x256! Cropping might occur!
[20:23:41]    gl: texture none size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture tunnel size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture brickback size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture brick1 size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture lava_red size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture water1 size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture water2 size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture water3 size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture water1 size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture acid size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture water size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture earth size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture earth_dry size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture earth_rough size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture earth_topsoil size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture earth_midsoil size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture ashes size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture sand_rough size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture ore size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture granite size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture gold size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture rock size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture rock_cracked size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture firestone size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture coal size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture sand_smooth size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture Ruby size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture Amethyst size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture ice2 size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture ice3 size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture snow1 size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture tunnel_NRM size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture brickback_NRM size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture brick1_NRM size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture earth_NRM size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture earth_dry_NRM size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture earth_rough_NRM size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture earth_topsoil_NRM size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture earth_midsoil_NRM size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture ashes_NRM size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture sand_rough_NRM size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture ore_NRM size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture granite_NRM size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture gold_NRM size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture rock_NRM size mismatch (512x512 vs 256x256)!
[20:23:41]    gl: texture rock_cracked_NRM size mismatch (512x512 vs 256x256)!
[20:23:42]    gl: texture firestone_NRM size mismatch (512x512 vs 256x256)!
[20:23:42]    gl: texture coal_NRM size mismatch (512x512 vs 256x256)!
[20:23:42]    gl: texture sand_smooth_NRM size mismatch (512x512 vs 256x256)!
[20:23:42]    gl: texture Ruby_NRM size mismatch (512x512 vs 256x256)!
[20:23:42]    gl: texture Amethyst_NRM size mismatch (512x512 vs 256x256)!
[20:23:42]    gl: texture ice2_NRM size mismatch (512x512 vs 256x256)!
[20:23:42]    gl: texture ice3_NRM size mismatch (512x512 vs 256x256)!
[20:23:42]    gl: texture snow1_NRM size mismatch (512x512 vs 256x256)!
[20:23:42]   gl: Texturing uses 32 slots at 256x256, 6 levels (22 MB total)
[20:23:43] Created 240x80 ambient map in 0.041 secs
[20:23:43] Placing environment objects...
[20:23:43] Game started.
terminate called after throwing an instance of 'std::bad_alloc'
  what():  std::bad_alloc
6.1 unix: Caught signal SIGABRT
./openclonk[0x7b1055]
/lib/x86_64-linux-gnu/libpthread.so.0(+0x10340)[0x7f7bd25ad340]
/lib/x86_64-linux-gnu/libc.so.6(gsignal+0x39)[0x7f7bcf447cc9]
/lib/x86_64-linux-gnu/libc.so.6(abort+0x148)[0x7f7bcf44b0d8]
/usr/lib/x86_64-linux-gnu/libstdc++.so.6(_ZN9__gnu_cxx27__verbose_terminate_handlerEv+0x155)[0x7f7bcfd52535]
/usr/lib/x86_64-linux-gnu/libstdc++.so.6(+0x5e6d6)[0x7f7bcfd506d6]
/usr/lib/x86_64-linux-gnu/libstdc++.so.6(+0x5e703)[0x7f7bcfd50703]
/usr/lib/x86_64-linux-gnu/libstdc++.so.6(+0x5e922)[0x7f7bcfd50922]
/usr/lib/x86_64-linux-gnu/libstdc++.so.6(_Znwm+0x7d)[0x7f7bcfd50e0d]
/usr/lib/x86_64-linux-gnu/libstdc++.so.6(_Znam+0x9)[0x7f7bcfd50ea9]
./openclonk(_ZN12C4FoWAmbient19UpdateFromLandscapeERK11C4LandscapeRK6C4Rect+0x2c0)[0x957e2e]
./openclonk(_ZN11C4Landscape10DoRelightsEv+0x174)[0x962626]
./openclonk(_ZN11C4Landscape4DrawER13C4TargetFacetP11C4FoWRegion+0xef)[0x962465]
./openclonk(_ZN10C4Viewport4DrawER13C4TargetFacetb+0x7a9)[0x85f297]
./openclonk(_ZN10C4Viewport7ExecuteEv+0x139)[0x85f8a3]
./openclonk(_ZN14C4ViewportList7ExecuteEb+0x6f)[0x86151d]
./openclonk(_ZN16C4GraphicsSystem7ExecuteEv+0x1bc)[0x85c19e]
./openclonk(_ZN12C4FullScreen13PerformUpdateEv+0x16)[0x840dce]
./openclonk(_ZN8C4Window13RequestUpdateEv+0x23)[0x7bee6f]
./openclonk(_ZN12C4FullScreen7ExecuteEv+0x40)[0x8409a4]
./openclonk(_ZN13C4Application4DrawEv+0x24)[0x83f680]
./openclonk(_ZN22C4ApplicationGameTimer7ExecuteEiP6pollfd+0x22c)[0x83fe20]
./openclonk(_ZN12StdScheduler15DoScheduleProcsEi+0x487)[0xb971ad]
./openclonk(_ZN13C4AbstractApp15DoScheduleProcsEi+0x33)[0xb0834b]
./openclonk(_ZN12StdScheduler13ScheduleProcsEi+0x234)[0xb957b0]
./openclonk(_ZN13C4AbstractApp3RunEv+0x1f)[0xb08307]
./openclonk(main+0x18f)[0x7b12a9]
/lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xf5)[0x7f7bcf432ec5]
./openclonk[0x7b0ccd]
```
After that, I rerun the game and started directly the mission  "Deadly Grotto" and It worked.

**Comment by frankstein-1** [comment]
The game will crash when you start the mission "the Raid", exit the mission, and start "Deadly Grotto".
This happen under Mint 17.1 rebbeca.
Does this problem exist under other platforms ?

**Comment by walachey** [comment]
Yes, I can reproduce it under windows. Thanks for reporting. I'll transfer the bug report to the official bugtracker so that it gets more attention. This seems to be a very urgent issue.

**Comment by aburgm** [comment]
This is fixed in master with eab0f28. Thanks for the report!
